<ui version="4.0" >
 <author>Aaron Racicot</author>
 <class>InterviewStart</class>
 <widget class="QDialog" name="InterviewStart" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>OpenOceanMap - Interview</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6" >
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4" >
         <item>
          <widget class="QGroupBox" name="interviewee_2" >
           <property name="font" >
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="title" >
            <string>Interviewer</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5" >
            <item>
             <widget class="QWidget" native="1" name="interviewee_first_name_2" >
              <layout class="QHBoxLayout" name="_3" >
               <property name="spacing" >
                <number>4</number>
               </property>
               <property name="margin" >
                <number>4</number>
               </property>
               <item>
                <widget class="QLabel" name="date_text" >
                 <property name="minimumSize" >
                  <size>
                   <width>180</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>180</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text" >
                  <string>1. Interview date (dd/mm/yyyy):</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="date_line" >
                 <property name="enabled" >
                  <bool>true</bool>
                 </property>
                 <property name="minimumSize" >
                  <size>
                   <width>150</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>150</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text" >
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" native="1" name="interviewee_first_name_8" >
              <layout class="QHBoxLayout" name="_23" >
               <property name="spacing" >
                <number>4</number>
               </property>
               <property name="margin" >
                <number>4</number>
               </property>
               <item>
                <widget class="QLabel" name="interviewer1_first_name_text_2" >
                 <property name="minimumSize" >
                  <size>
                   <width>180</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>180</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text" >
                  <string>2. Interviewer first name:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="interviewer1_first_name_line" >
                 <property name="enabled" >
                  <bool>true</bool>
                 </property>
                 <property name="minimumSize" >
                  <size>
                   <width>150</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>150</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text" >
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" native="1" name="interviewee_first_name_9" >
              <layout class="QHBoxLayout" name="_24" >
               <property name="spacing" >
                <number>4</number>
               </property>
               <property name="margin" >
                <number>4</number>
               </property>
               <item>
                <widget class="QLabel" name="interviewer1_first_name_text_3" >
                 <property name="minimumSize" >
                  <size>
                   <width>180</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>180</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text" >
                  <string>3. Interviewer last name:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="interviewer1_last_name_line" >
                 <property name="enabled" >
                  <bool>true</bool>
                 </property>
                 <property name="minimumSize" >
                  <size>
                   <width>150</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>150</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text" >
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" native="1" name="interviewee_first_name_3" >
              <layout class="QHBoxLayout" name="_8" >
               <property name="spacing" >
                <number>4</number>
               </property>
               <property name="margin" >
                <number>4</number>
               </property>
               <item>
                <widget class="QLabel" name="interviewer1_first_name_text" >
                 <property name="minimumSize" >
                  <size>
                   <width>180</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>180</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text" >
                  <string>4. Interviewer 2 first name:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="interviewer2_first_name_line" >
                 <property name="enabled" >
                  <bool>true</bool>
                 </property>
                 <property name="minimumSize" >
                  <size>
                   <width>150</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>150</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text" >
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" native="1" name="interviewee_first_name_4" >
              <layout class="QHBoxLayout" name="_9" >
               <property name="spacing" >
                <number>4</number>
               </property>
               <property name="margin" >
                <number>4</number>
               </property>
               <item>
                <widget class="QLabel" name="interviewer1_last_name_text" >
                 <property name="minimumSize" >
                  <size>
                   <width>180</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>180</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text" >
                  <string>5. Interviewer 2 last name:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="interviewer2_last_name_line" >
                 <property name="enabled" >
                  <bool>true</bool>
                 </property>
                 <property name="minimumSize" >
                  <size>
                   <width>150</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>150</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text" >
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" native="1" name="interviewee_first_name_5" >
              <layout class="QHBoxLayout" name="_10" >
               <property name="spacing" >
                <number>4</number>
               </property>
               <property name="margin" >
                <number>4</number>
               </property>
               <item>
                <widget class="QLabel" name="mood_text" >
                 <property name="minimumSize" >
                  <size>
                   <width>180</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>180</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text" >
                  <string>6. Interviewer mood:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="mood_line" >
                 <property name="enabled" >
                  <bool>true</bool>
                 </property>
                 <property name="minimumSize" >
                  <size>
                   <width>150</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>150</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text" >
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" native="1" name="interviewee_first_name_7" >
              <layout class="QHBoxLayout" name="_13" >
               <property name="spacing" >
                <number>4</number>
               </property>
               <property name="margin" >
                <number>4</number>
               </property>
               <item>
                <widget class="QLabel" name="mood_text_3" >
                 <property name="minimumSize" >
                  <size>
                   <width>180</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>180</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text" >
                  <string>7. Interview location:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="location_line" >
                 <property name="enabled" >
                  <bool>true</bool>
                 </property>
                 <property name="minimumSize" >
                  <size>
                   <width>150</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>150</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text" >
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" native="1" name="interviewee_first_name_6" >
              <layout class="QHBoxLayout" name="_12" >
               <property name="spacing" >
                <number>4</number>
               </property>
               <property name="margin" >
                <number>4</number>
               </property>
               <item>
                <widget class="QLabel" name="mood_text_2" >
                 <property name="minimumSize" >
                  <size>
                   <width>180</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>180</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text" >
                  <string>8. Data collection method:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="dataMethod_comboBox" >
                 <property name="maximumSize" >
                  <size>
                   <width>150</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <item>
                  <property name="text" >
                   <string/>
                  </property>
                 </item>
                 <item>
                  <property name="text" >
                   <string>electronic</string>
                  </property>
                 </item>
                 <item>
                  <property name="text" >
                   <string>hard copy</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer" >
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="vessel_info" >
       <property name="font" >
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="title" >
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout" >
        <item>
         <widget class="QWidget" native="1" name="home_port_13" >
          <property name="font" >
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <layout class="QHBoxLayout" name="_22" >
           <property name="spacing" >
            <number>4</number>
           </property>
           <property name="margin" >
            <number>4</number>
           </property>
           <item>
            <widget class="QLabel" name="years_text_6" >
             <property name="minimumSize" >
              <size>
               <width>170</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>170</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font" >
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text" >
              <string>9. Fisher registration #:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="fish_regis_line" >
             <property name="enabled" >
              <bool>true</bool>
             </property>
             <property name="minimumSize" >
              <size>
               <width>180</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>180</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font" >
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text" >
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" native="1" name="home_port_8" >
          <property name="font" >
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <layout class="QHBoxLayout" name="_17" >
           <property name="spacing" >
            <number>4</number>
           </property>
           <property name="margin" >
            <number>4</number>
           </property>
           <item>
            <widget class="QLabel" name="landing_port_text_3" >
             <property name="minimumSize" >
              <size>
               <width>170</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>170</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font" >
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text" >
              <string>10. Years fishing experience:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="yrs_fishing_line" >
             <property name="enabled" >
              <bool>true</bool>
             </property>
             <property name="minimumSize" >
              <size>
               <width>180</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>180</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font" >
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text" >
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" native="1" name="button_box" >
          <layout class="QHBoxLayout" name="_19" >
           <property name="spacing" >
            <number>2</number>
           </property>
           <property name="margin" >
            <number>2</number>
           </property>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pbnCancel" >
             <property name="font" >
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text" >
              <string>Cancel</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer" >
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType" >
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>50</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pbnSelectFishery" >
             <property name="font" >
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text" >
              <string>Continue</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>date_line</tabstop>
  <tabstop>interviewer1_first_name_line</tabstop>
  <tabstop>interviewer1_last_name_line</tabstop>
  <tabstop>interviewer2_first_name_line</tabstop>
  <tabstop>interviewer2_last_name_line</tabstop>
  <tabstop>mood_line</tabstop>
  <tabstop>location_line</tabstop>
  <tabstop>dataMethod_comboBox</tabstop>
  <tabstop>fish_regis_line</tabstop>
  <tabstop>yrs_fishing_line</tabstop>
  <tabstop>pbnSelectFishery</tabstop>
  <tabstop>pbnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
